src/cryptoadvance/specter/templates/includes/tool-tip.html

```diff
@@ -1,110 +1,181 @@
+        .paragraph {
+            display: block;
+            text-align: start;
+            font-size: 13px;
+            font-weight: 300;
+            line-height: 1.3;
+            color: white;
         }
-
-        a:focus + .tool-tip_basic .tool-tip__info {
-            display: inline-block;
+        .tooltip:hover .tooltip__box {
+            display: flex;
+            flex-direction: column;
         }
         @media (hover: none) {
-            .tool-tip {
+            .tooltip {
                 display: none;
             }
         }
     </style>
 
-    <div class="tool-tip">
-        <i class="tool-tip__icon">i</i>
-        <div class="tool-tip__info" id="tool-tip-box">
-            <slot name="title" class="info__title"></slot>
+    <div class="tooltip">
+        <img class="tooltip__icon" src="{{ url_for('static', filename='img/light-bulb.svg') }}" id="tooltip-img"></img>
+        <div class="tooltip__box" id="tooltip-box">
+            <slot name="title" class="title"></slot>
             <slot name="paragraph" class="paragraph"></slot>
         </div>
     </div>
-    
 </template>
   
 <script type="module">
     class TooltipElement extends HTMLElement {
         constructor() {
             super();
             var shadowRoot = this.attachShadow({mode: 'open'});
-            var template = document.getElementById('tool-tip-template-id');
+            var template = document.getElementById('tooltip-template-id');
             var clone = template.content.cloneNode(true);
-            this.tooltipBox = clone.getElementById("tool-tip-box")
-            this.tooltipBox.style.width = this.getAttribute("width");
+            this.tooltipImg = clone.getElementById('tooltip-img');
+            this.tooltipDiv = clone.querySelector('.tooltip');
+            // CustomEvent that can be picked up to adjust flex properties in the parent
+            this.tooltipDiv.addEventListener("mouseover", () => {
+                let event = new CustomEvent('customhover');
+                this.dispatchEvent(event);
+            })
+            this.tooltipDiv.addEventListener("mouseout", () => {
+                let event = new CustomEvent('customout');
+                this.dispatchEvent(event);
+            })
+            this.tooltipBox = clone.getElementById("tooltip-box");
             shadowRoot.appendChild(clone);
         }
+
+        static get observedAttributes() {
+            return ['width', 'move'];
+        }
+
+        connectedCallback() {
+            if (this.getAttribute("width")) {
+                this.tooltipBox.style.width = this.getAttribute("width");
+            }
+            if (this.getAttribute("move-right")) {
+                this.tooltipBox.style.left = "60px";
+            }
+            if (this.getAttribute("image")) {
+                if (this.getAttribute("image") == "note") {
+                    this.tooltipImg.src = "{{ url_for('static', filename='img/note.svg') }}";
```

Comment:
shouldn't we allow user to supply image url, instead of a flag for just 1 image?

Reply:
We could, but there needs to be a matching image in the static folder. My idea was to enable more pictures via this if construct over time. Might also be the case that we don't need more than the two.

Reply:
Ahh, got it